src/QueryReflection/QueryReflection.php

```diff
@@ -306,7 +306,7 @@ private function replaceParameters(string $queryString, array $parameters): stri
         foreach ($parameters as $placeholderKey => $parameter) {
             $value = $parameter->simulatedValue;
 
-            if (\is_string($value)) {
+            if (\is_string($value) && !is_numeric($value)) {
```

Comment:
not sure if this is the right approach here, as it would not escape numeric values at all now :thinking: 
And at least one test expects that numeric values in the WHERE clause are escaped

Another approach would be to strip the limit and offset of queries here too